crates/types/src/vid.rs

```diff
@@ -231,6 +231,12 @@ impl VidScheme for VidSchemeType {
     fn get_multiplicity(common: &Self::Common) -> u32 {
         <Advz as VidScheme>::get_multiplicity(common)
     }
+
+    /// Helper function for testing only
+    #[cfg(feature = "test-srs")]
```

Comment:
I don't think we ever run tests with `test-srs` -- it doesn't seem to even be in the justfile. Is this just for downstream users like the sequencer?

Reply:
Oops, ignore this one -- I see it was used in the test which was also feature gated

Reply:
Yes, that's intentional that we never run with this feature.  It is a feature to match the 'test-srs' feature in jellyfish.